When failing it prints out the message below. I have been tripped up by this more than once, and it also confused some of our Outreachy applicants. It seems running once to see the message then running again with `GITVER=xxxxxx make html` is a small enough price to pay to make sure you know what you are doing.
```
$ make html
installed numpy db786d9fb1 != current repo git version 01fb828e84
set GITVER to db786d9fb1 to override
Makefile:53: recipe for target 'version-check' failed
make: *** [version-check] Error 1
```

I will extend the test to always match `Unknown`

--- a/doc/Makefile
+++ b/doc/Makefile
@@ -28,7 +28,7 @@ ALLSPHINXOPTS   = -WT --keep-going -d build/doctrees $(PAPEROPT_$(PAPER)) \
   $(SPHINXOPTS) source
 
 .PHONY: help clean html web pickle htmlhelp latex changes linkcheck \
-        dist dist-build gitwash-update
+        dist dist-build gitwash-update version-check
 
 #------------------------------------------------------------------------------
 
@@ -53,7 +53,7 @@ ifeq "$(GITVER)" "Unknown"
 	# @echo sdist build with unlabeled sources
 else ifneq ($(NUMPYVER),$(GITVER))
 	@echo installed numpy $(NUMPYVER) != current repo git version \'$(GITVER)\'
-	@echo set GITVER to $(NUMPYVER) to override
+	@echo use '"make dist"' or '"set GITVER=$(NUMPYVER) make $(MAKECMDGOALS) ..."'
 	@exit 1
 else
 	# for testing

--- a/doc/Makefile
+++ b/doc/Makefile
@@ -49,8 +49,10 @@ clean:
 	-rm -rf build/* source/reference/generated
 
 version-check:
-ifneq ($(NUMPYVER),$(GITVER))
-	@echo installed numpy $(NUMPYVER) != current repo git version $(GITVER)
+ifeq "$(GITVER)" "Unknown"
+	# @echo sdist build with unlabeled sources
+else ifneq ($(NUMPYVER),$(GITVER))
+	@echo installed numpy $(NUMPYVER) != current repo git version \'$(GITVER)\'
 	@echo set GITVER to $(NUMPYVER) to override
 	@exit 1
 else

